src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -1085,6 +1085,13 @@ void QFieldCloudProjectsModel::refreshProjectDeltaList( const QString &projectId
 
   NetworkReply *deltaStatusReply = mCloudConnection->get( QStringLiteral( "/api/v1/deltas/%1/" ).arg( mCloudProjects[index].id ) );
 
+  if ( mCloudProjects[index].deltaListModel )
+  {
+    delete mCloudProjects[index].deltaListModel;
+    mCloudProjects[index].deltaListModel = nullptr;
```

Comment:
Could this be converted to a `std::unique_ptr`?

Reply:
I just checked, the compiler complains a lot. I guess because it is part of a `struct` that is part of `QList`.

Reply:
Then just let it be